### DISCUSSION TITLE
NDI Source not detected by DistroAV

### DISCUSSION THREAD

**Discussion by DanielFrankDK** [DistroAV/DistroAV#1169] [Q&A - Support] [answered]
I'm having some issues using DistroAV. My one and only NDI source, which works just fine through NDI Studio Monitor, isn't being discovered by DistroAV.

Studio Monitor:

![image](https://github.com/user-attachments/assets/0a8073cd-c34e-4b7d-a277-b3db9ac08236)

DistroAV / OBS:

![image](https://github.com/user-attachments/assets/15bdc485-b51b-44fb-a538-636bbc855b8c)

The source "SVHS (FULL)" is not on the list.

I've installed latest NDI Runtime / Tools, latest version of DistroAV. What am I doing wrong?

I should mention that the NDI source is not enlisted in a Discovery Service Network, but that shouldn't be a problem.

**Comment by RayneYoruka** [comment, marked as answer]
This seems like firewall https://github.com/DistroAV/DistroAV/wiki/2.-Troubleshooting#firewall-settings--requirements

**Reply by DanielFrankDK** [reply]
Well, butter my biscuit! I'll be damned. That did the trick!

My firewall was already set up to "Private Network", yet I had to manually add OBS Studio in the firewall rules.

Thank you very much!

**Reply by felipefalkiner** [reply]
Even if my Windows Firewall is disabled?

**Reply by RayneYoruka** [reply]
If it's not your machine it's something else in your network. You'll have to find out on your own unfortunately.

**Reply by felipefalkiner** [reply]
> If it's not your machine it's something else in your network. You'll have to find out on your own unfortunately.

Got it, thanks, will check out.
Fun fact is that other software as Resolume Arena can find those NDI Sources.

**Reply by RayneYoruka** [reply]
Those are common symptoms of obs not allowed in the firewall. It's within the troubleshooting page.

**Reply by felipefalkiner** [reply]
> Those are common symptoms of obs not allowed in the firewall. It's within the troubleshooting page.

Yeah, I had rules to enable OBS, the installer now does this also, I've just deleted all the rules and it's working again hahaha. The firewall as disabled but it was still affecting, crazy.
